icons/copy-equal.svg

```diff
@@ -0,0 +1,16 @@
+<svg
```

Comment:
What is the use case exactly of this icon?

Reply:
I was just aiming for all `circle-*`, `square-*` and `copy-*` (essentially just multiple squares) icons to be more of a set with the same set of elements…

Reply:
Ah check, I get your thinking.
But I'm not sure about the use case. Let's keep this one out if we couldn't come up with a use case.